src/test/java/net/minecraftforge/test/TestCapabilityMod.java

```diff
@@ -47,7 +47,7 @@ public void preinit(FMLPreInitializationEvent evt)
     @SubscribeEvent
     public void onInteract(PlayerInteractEvent.LeftClickBlock event)
     {
-        if (event.getItemStack() == null) return;
+        if (event.getItemStack().func_190926_b()) return;
```

Comment:
It's a silly test in the first place, I'll leave it this way in case they remove the STICK requirement for testing later.